MAINTAINERS

```diff
@@ -1,2 +1,7 @@
+Andrey Dyachkov <andrey.dyachkov@zalando.de>
 Dmitry Sorokin <dmitriy.sorokin@zalando.de>
-
+Ricardo De Cillo <ricardo.de.cillo@zalando.de>
+Valentine Gogichashvili <valentine.gogichashvili@zalando.de>
+Vyacheslav Stepanov <vyacheslav.stepanov@zalando.de>
+Lionel Montrieux <lionel.montrieux@zalando.de>
+Robert Garrett <robert.garrett@zalando.de>
```

Comment:
@lmontrieux Can you please remove some file parts or close pr if it is not needed?

Reply:
oooh, this is an old one :). Yes, I'll have a look and close the PR if it's not needed anymore